Util/Xamarin.Build.Download/source/Xamarin.Build.Download.Tests/HashTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Xamarin.Build.Download;
+using Xunit;
+
+namespace NativeLibraryDownloaderTests
+{
+    public class HashTests
+    {
+        [Theory]
+        [InlineData("C:\\path\\to\\file.aar")]
```

Comment:
This will fail on mac with invalid paths. I think?

Reply:
The intent is just to see that i get no funny characters (just numbers, and digits) in the CRC code returned.  I'll add a test case for a mac path too for kicks.